### PR TITLE
Onceler Github Action deployment

### PR DESCRIPTION
**Pull request by boris-ning-usds**
This change set allows us to automatically deploy via github actions - https://github.com/usds/onceler/actions.
- Added a simple script and manifest file for github action to deploy on push to main branch.
- Cleaned up gitignore to avoid checking in operating system specifics.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          bundler-cache: false
+
+      - name: Deploy to cloud.gov
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          cf_api: ${{ secrets.CF_API }}
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: ${{ secrets.CF_ORG }}
+          cf_space: ${{ secrets.CF_SPACE }}
```
Comment (TomNUSDS) [inline .github/workflows/deploy.yml:25]: Are you keeping the org and space "secret" to keep it generic or are they considered sensitive?

**Review by TomNUSDS — APPROVED** [review]
LGTM. 
Maybe a README.md to discuss the github configuration at some point?

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (boris-ning-usds) [reply .github/workflows/deploy.yml:25]: I consider them sensitive, this is a public repository after all.

**Comment by boris-ning-usds** [comment]
Yeah, we probably need an engineering discussion on maintenance of infrastructure applications.

**Pull request merged**